test/fuzz/README.md

```diff
@@ -5,33 +5,35 @@ The fuzz tests in this repo send random transactions to the chain in order to tr
 Example invocation:
 
 ```bash
-FUZZ_TEST=TRUE SEED=1 RPC_MODE="SingleRpc" \
-    RPC_URLS="http://localhost:26657" \
-    MAX_ITERATIONS=100 MODE="alternate" \
-    NUM_ACTORS=12 EPOCH_LENGTH=14 \
-    /usr/bin/go test -timeout 15m -run ^TestFuzzTestSuite$ -v ./test/fuzz
+FUZZ_TEST=TRUE go test -timeout 15m -run ^TestFuzzTestSuite$ -v ./test/fuzz
```

Comment:
If you just run this against a basic, `allorad start` network, it won't work (complains about not having a `faucet` acct, and I guess about others too?). 
Since these requirements are not specified, this should explain how to run it against the local network that this is configured to run against.

```suggestion
bash test/local_testnet_l1.sh
FUZZ_TEST=TRUE go test -timeout 15m -run ^TestFuzzTestSuite$ -v ./test/fuzz
```

Reply:
ah good point thank you